src/web/screens/accountInformation.tsx

```diff
@@ -49,13 +32,12 @@ function AccountInformation() {
         </h3>
         <span>{participantTypes}</span>
       </div>
-      <AccountInformationFooter />
     </>
   );
 }
 
 export const AccountInformationRoute: PortalRoute = {
-  description: 'General Account Info.',
+  description: 'Participant Information',
```

Comment:
hm I had a different idea of what the Participant Information screen is. I have asked for clarification here: https://thetradedesk.slack.com/archives/C04SFSESXQF/p1689822600262929

Reply:
seems like what you have done is right. I'll rename my new screen for now.